docs/apache-airflow/howto/operator/postgres_operator_howto_guide.rst

```diff
@@ -0,0 +1,197 @@
+        create_pet_table = PostgresOperator(
+                                            "create_pet_table",
+                                            postgres_conn_id = "postgres_default",
+                                            sql = """
+                                            CREATE TABLE IF NOT EXISTS pet (
+                                                  pet_id SERIAL PRIMARY KEY,
+                                                  name VARCHAR NOT NULL,
+                                                  pet_type VARCHAR NOT NULL,
+                                                  birth_date DATE NOT NULL,
+                                                  OWNER VARCHAR NOT NULL);
+                                                  """
+                                            )
+
+
+Dumping SQL statements into your PostgresOperator isn't quite appealing and will create maintainability pains somewhere
+down to the road. To prevent this, Airflow offers an elegant solution. This is how it works: you simply create
+a directory inside the DAG folder called ``sql`` and then put all the SQL files containing your SQL queries inside it.
+
+Your ``dags/sql/pet_schema.sql`` should like this:
+
+::
+
+      -- create pet table
+      CREATE TABLE IF NOT EXISTS pet (
+          pet_id SERIAL PRIMARY KEY,
+          name VARCHAR NOT NULL,
+          pet_type VARCHAR NOT NULL,
+          birth_date DATE NOT NULL,
+          OWNER VARCHAR NOT NULL);
+
+
+Now let's refactor ``create_pet_table`` in our DAG:
+
+.. code-block:: python
+
+        create_pet_table = PostgresOperator(
+                                            "create_pet_table",
+                                            postgres_conn_id = "postgres_default",
+                                            sql = "sql/pet_schema.sql"
+                                            )
+
+
+Inserting data into a Postgres database table
+---------------------------------------------
+
+Let's say we already have the SQL insert statement below in our ``dags/sql/pet_schema.sql`` file:
+
+::
+
+  -- populate pet table
+  INSERT INTO pet VALUES ( 'Max', 'Dog', '2018-07-05', 'Jane');
+  INSERT INTO pet VALUES ( 'Susie', 'Cat', '2019-05-01', 'Phil');
+  INSERT INTO pet VALUES ( 'Lester', 'Hamster', '2020-06-23', 'Lily');
+  INSERT INTO pet VALUES ( 'Quincy', 'Parrot', '2013-08-11', 'Anne');
+
+We can then create a PostgresOperator task that populate the ``pet`` table.
+
+.. code-block:: python
+
+  populate_pet_table = PostgresOperator(
+                                        "populate_pet_table",
+                                        postgres_conn_id = "postgres_default",
+                                        sql = "sql/pet_schema.sql"
+                                        )
+
+
+Fetching records from your postgres database table
+--------------------------------------------------
+
+Fetching records from your postgres database table can be as simple as:
```

Comment:
I think that it's important to comment that the result of the SELECT command will not be returned as a xcom variable, for example. The execute method from the PostgresOperator class will just print the result in the log.

I saw this behavior last week trying to pull a SELECT result from a previous OracleOperator's task and just check that the same occurs to PostgresOperator.

https://stackoverflow.com/a/54101400/5499208